doc/rest.md

```diff
@@ -6,11 +6,27 @@ import {Rest} from 'spoonx/aurelia-api';
 
 ----------
 
-## .request(method, path[, body][, options])
+## Properties
```

Comment:
Please direct these changes to [the new docs](http://aurelia-api.spoonx.org/api_rest.html) :)